src/containers/collectibles/components/CollectiblesPage.module.css

```diff
@@ -0,0 +1,528 @@
+  width: 200px;
+  height: 200px;
+}
+
+.playIcon {
+  position: absolute;
+  top: 80px;
+  left: calc(50% - 24px);
+  width: 48px;
+  height: 48px;
+}
+
+.playIcon circle {
+  fill: none;
+}
+
+.playIcon path {
+  fill: #fffa;
+}
+
+.stamp {
+  position: absolute;
+  top: 182px;
+  left: 20px;
+  z-index: 10000;
+  font-size: var(--font-2xs);
+  color: var(--white);
+}
+
+.created {
+  padding: 2px 6px;
+  border-radius: 50px;
+  border: 1px solid var(--white);
+  background: var(--primary);
+}
+
+.owned {
+  padding: 2px 6px;
+  border-radius: 50px;
+  border: 1px solid var(--white);
+  background: var(--secondary);
+}
+
+.nftTitle {
+  padding: 10px 0 12px 0;
+  width: 100%;
+  text-align: center;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+.detailsContainer {
+  width: 216px;
+  margin: 8px;
+}
+
+.detailsContainer:hover {
+  box-shadow: 0 1px 5px 1px var(--tile-shadow-1-alt), 0 1px 0 0 var(--tile-shadow-2), 0 2px 10px -2px var(--tile-shadow-3);
+  border-radius: 8px;
+}
+
+.detailsContainer > div {
+  width: inherit;
+}
+
+.mobileDetailsContainer {
+  display: flex;
+  flex-grow: 1;
+}
+
+.mobileDetailsContainer > div {
+  flex-grow: 1;
+}
+
+.details {
+  margin: 56px 24px 24px 0;
+  width: 312px;
+}
+
+.detailsTitle {
+  margin-bottom: 20px;
+  font-size: var(--font-2xl);
+  line-height: 32px;
+}
+
+.detailsStamp {
+  margin-bottom: 20px;
+  font-size: var(--font-2xs);
+  color: var(--white);
+}
+
+.mobileDetailsStamp {
+  margin-top: 28px;
+}
+
+.date {
+  margin-top: 8px;
+  margin-bottom: 20px;
+  font-weight: var(--font-medium);
+}
+
+.mobileDateWrapper {
+  display: flex;
+  margin: 8px 0;
+}
+
+.mobileDate {
+  margin: 0 8px;
+}
+
+.detailsStamp .created, .detailsStamp .owned {
+  border: none;
+}
+
+.detailsDescription {
+  margin-bottom: 16px;
+  font-weight: var(--font-medium);
+  line-height: 125%;
+}
+
+.mobileDetailsDescription {
+  margin-top: 16px;
+  margin-bottom: 24px;
+}
+
+.link {
+  display: flex;
+  align-items: center;
+  color: var(--secondary);
+  text-decoration: underline !important;
+}
+
+.link:hover {
+  color: var(--secondary);
+}
+
+.linkIcon {
+  width: 16px;
+  height: 16px;
+  margin-right: 6px;
+}
+
+.linkIcon path {
+  fill: var(--secondary);
+}
+
+
+.volumeIcon {
+  position: absolute;
+  bottom: 8px;
+  right: 8px;
+  width: 32px;
+  height: 32px;
+  padding: 6px;
+  border: 2px solid var(--neutral-light-5);
+  border-radius: 100%;
+}
+
+.volumeIcon path {
+  fill: var(--neutral-light-5);
+}
+
+.modalHeader {
+  margin: 0 16px;
+}
+
+.modalTitle {
+  color: var(--neutral);
+  font-size: var(--font-l);
+}
+
+.modalBody {
+  width: 100%;
+  max-width: fit-content;
+  border-radius: 6px;
+  background-color: var(--neutral-light-10);
+  overflow-y: scroll;
+  overflow-x: hidden;
+  -webkit-overflow-scrolling: touch;
+}
+
+.modalMediaWrapper {
+  position: relative;
+  margin: 24px;
+  width: 342px;
+  height: 342px;
+  max-width: 512px;
+  max-height: 512px;
+}
+
+video {
```

Comment:
Since video is a native html element, I would with this within a selector ie. `.container video` just in case.